app/models/container_deployment.rb

```diff
@@ -6,5 +6,314 @@ class ContainerDeployment < ApplicationRecord
   has_many :container_volumes, :as => :parent
   has_many :custom_attributes, :as => :resource, :dependent => :destroy
   has_many :authentications, :as => :resource, :dependent => :destroy
-  serialize :customize, Hash
+  serialize :customizations, Hash
+  AUTH_CONSTS = {"AuthenticationAllowAll" => AuthenticationAllowAll, "AuthenticationHtpasswd" => AuthenticationHtpasswd, "AuthenticationLdap" => AuthenticationLdap, "AuthenticationRequestHeader" => AuthenticationRequestHeader, "AuthenticationOpenId" => AuthenticationOpenId, "AuthenticationGoogle" => AuthenticationGoogle, "AuthenticationGithub" => AuthenticationGithub, "AuthPrivateKey" => AuthPrivateKey, "AuthenticationRhsm" => AuthenticationRhsm}.freeze
+  AUTHENTICATIONS_NAMES = {"AuthenticationAllowAll" => "AllowAllPasswordIdentityProvider", "AuthenticationHtpasswd" => "HTPasswdPasswordIdentityProvider", "AuthenticationLdap" => "LDAPPasswordIdentityProvider", "AuthenticationRequestHeader" => "RequestHeaderIdentityProvider", "AuthenticationOpenId" => "OpenIDIdentityProvider", "AuthenticationGoogle" => "GoogleIdentityProvider", "AuthenticationGithub" => "GitHubIdentityProvider", "AuthPrivateKey" => "ssh", "AuthenticationRhsm" => "rhsm"}.freeze
```

Comment:
Is the AUTHENTICATIONS_NAMES stuff really just a property of the Authentication subclasses, or are those things very specific to container deployment?  I ask because the `to_ansible_config_format` is on each of the models, even though those seem *very* container deployment specific.